packages/plugins/ai/test/schedule.test.ts

```
@@ -0,0 +1,390 @@
+						confidence: 0.95,
+					}),
+				},
+			}],
+		}), { status: 200, headers: { 'Content-Type': 'application/json' } }));
+
+		const slot = await scheduleAI('Schedule 1 hour review session');
+
+		expect(slot).toBeInstanceOf(Interval);
+		expect(slot.constructor).toBe(Interval);
+		expect(slot.constructor === Interval).toBe(true);
+		expect(slot.contains === slot.contains).toBe(true);
```

Biome (2.5.6)

[error] This comparison uses the same expression on both sides.

(lint/suspicious/noSelfCompare)